.github/workflows/shippable_builds.yml

```diff
@@ -697,11 +740,14 @@ jobs:
           APPLICATION_ID: ${{ steps.pkginfo.outputs.APPLICATION_ID }}
           REPO: ${{ github.repository }}
           APP_SHA: ${{ steps.shanotes.outputs.app_sha }}
+          RELEASE_TYPE: ${{ vars.RELEASE_TYPE }}
         run: |
           # r0adkll/upload-google-play expects the release notes in a different structure
-          FILEPATH=app-metadata/${APPLICATION_ID}/en-US/changelogs/${VERSION_CODE}.txt
           mkdir whatsnew
-          wget -O whatsnew/whatsnew-en-US https://raw.githubusercontent.com/${REPO}/${APP_SHA}/${FILEPATH}
+          if [[ "$RELEASE_TYPE" = "beta" || "$RELEASE_TYPE" = "release" ]]; then
+            FILEPATH=app-metadata/${APPLICATION_ID}/en-US/changelogs/${VERSION_CODE}.txt
+            wget -O whatsnew/whatsnew-en-US https://raw.githubusercontent.com/${REPO}/${APP_SHA}/${FILEPATH}
+          fi
 
       - name: Publish to Google Play
         id: publish_play
```

Comment:
While you are here, could you adjust the release name for the play store upload to be `${{ versionCode }} (${{ versionName }})`? This makes it more consistent with the auto-generated name.